lib/billy/proxy_connection.rb

```diff
@@ -74,12 +74,25 @@ def handle_request
 
     private
 
+    def prepare_response_headers_for_evma_httpserver(headers)
+      # Remove the headers below because they will be added later by evma_httpserver (EventMachine::DelegatedHttpResponse).
+      # See https://github.com/eventmachine/evma_httpserver/blob/master/lib/evma_httpserver/response.rb
+      headers_to_remove = [
+        'transfer-encoding',
+        'content-length',
+        'content-encoding'
```

Comment:
My comment here got collapsed.  I don't see this header being added later.  Why are we deleting here?